src/desktop/package.json

```diff
@@ -168,8 +172,31 @@
     "react-router-dom": "5.0.0",
     "react-transition-group": "4.0.1",
     "realm": "2.23.0",
-    "recharts": "1.6.0",
+    "recharts": "1.6.1",
     "redux": "4.0.1",
     "usb": "1.5.0"
+  },
+  "jest": {
+    "runner": "@jest-runner/electron",
+    "globalSetup": "<rootDir>scripts/jestSetup.js",
+    "globalTeardown": "<rootDir>scripts/jestTeardown.js",
+    "testEnvironment": "<rootDir>scripts/jestEnv.js",
```

Comment:
I'm trying to set up the puppeteer environment in a docker container for Buildkite but I'm running into some sandbox issues with Chromium. Could we disable the end-to-end tests for now and just run the unit tests in an environment that doesn't use puppeteer?

Reply:
Can you try to disable sandbox for Puppeteer at `src/desktop/scripts/jestSetup.js` by adding args to `puppeteer.launch({args: ['--no-sandbox', '--disable-setuid-sandbox']})` and see if that works? As we fully trust the content and it's for tests only, I don't think there's a security issue by disabling the sandbox.

Reply:
I tried that as well and ran into another error about a display 🤔 I don't remember the exact text of the warning but I ran into a lot of the issues posted in https://github.com/GoogleChrome/puppeteer/issues/290